resources/migrations/000_migrations.yaml

```diff
@@ -15173,6 +15173,14 @@ databaseChangeLog:
               column:
                 name: action_id
 
+  - changeSet:
+      id: v48.00-007
```

Comment:
pretty sure 007 has been claimed, you need to use a different one

Reply:
🤦  I feel so dumb right now for not checking this